pages/chaoss/visualizations/contrib_importance.py

```diff
@@ -0,0 +1,251 @@
+from dash import html, dcc, callback
+import dash
+from dash import dcc
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import pandas as pd
+import logging
+from dateutil.relativedelta import *  # type: ignore
+import plotly.express as px
+from pages.utils.graph_utils import get_graph_time_values, color_seq
+from queries.contributors_query import contributors_query as ctq
+import io
+from cache_manager.cache_manager import CacheManager as cm
+from pages.utils.job_utils import nodata_graph
+import time
+import random 
+
+PAGE = "chaoss"  
+VIZ_ID = "contrib-importance"  
```

Comment:
change to "contrib-importance-pie" or something like it to make the naming descriptively distinct to the line graph (same for the file name)